scene/3d/audio_stream_player_3d.cpp

```diff
@@ -273,6 +273,7 @@ void AudioStreamPlayer3D::_notification(int p_what) {
 				AudioServer::get_singleton()->start_playback_stream(setplayback, bus_map, setplay.get(), actual_pitch_scale, linear_attenuation, attenuation_filter_cutoff_hz);
 				setplayback.unref();
 				setplay.set(-1);
+				_update_panning();
```

Comment:
Add a comment to explain why this is done:
```suggestion
				// Call `_update_panning()` immediately after starting the stream.
				// This re-sets bus levels to ensure they are correct for the  first few frames.
				// (These levels may be initialized incorrectly due to the call to start_playback_stream
				// ignoring the Area3D's bus options).
				_update_panning();
```